docs/making-yomitan-dictionaries.md

```diff
@@ -0,0 +1,106 @@
+| [`dictionary-term-bank-v3-schema.json`](https://github.com/themoeway/yomitan/tree/master/ext/data/schemas/dictionary-term-bank-v3-schema.json)             | `term_bank_${number}.json`       | Stores dictionary readings, definitions, etc.                                          |
+| [`dictionary-term-meta-bank-v3-schema.json`](https://github.com/themoeway/yomitan/tree/master/ext/data/schemas/dictionary-term-meta-bank-v3-schema.json)   | `term_meta_bank_${number}.json`  | Stores meta information about terms, such as frequency data and pitch accent data.     |
+
+## Packaging A Dictionary
+
+A dictionary can contain various types of information within the zip file. After creating an `index.json` and the relevant data files, zip them with all data `.json` files in the root directory of the zip, not in subfolders. Use the highest compression level possible to reduce the size.
+
+## Examples
+
+- The [official test dictionary](https://github.com/themoeway/yomitan/tree/master/test/data/dictionaries/valid-dictionary1) showcases the full range of features available in Yomitan dictionaries.
+
+## Schema Validation
+
+To validate schemas, configure [VSCode](https://code.visualstudio.com/docs/languages/json#_json-schemas-and-settings) to validate schemas or use a website such as [jsonschemavalidator](https://www.jsonschemavalidator.net/).
+
+For VSCode validation, use the following settings JSON:
+
+```json
+  "json.schemas": [
+    {
+      "fileMatch": ["kanji_bank_*.json"],
+      "url": "https://github.com/themoeway/yomitan/raw/master/ext/data/schemas/dictionary-kanji-bank-v3-schema.json"
+    },
+    {
+      "fileMatch": ["kanji_meta_bank_*.json"],
+      "url": "https://github.com/themoeway/yomitan/raw/master/ext/data/schemas/dictionary-kanji-meta-bank-v3-schema.json"
+    },
+    {
+      "fileMatch": ["tag_bank_*.json"],
+      "url": "https://github.com/themoeway/yomitan/raw/master/ext/data/schemas/dictionary-tag-bank-v3-schema.json"
+    },
+    {
+      "fileMatch": ["term_bank_*.json"],
+      "url": "https://github.com/themoeway/yomitan/raw/master/ext/data/schemas/dictionary-term-bank-v3-schema.json"
+    },
+    {
+      "fileMatch": ["term_meta_bank_*.json"],
+      "url": "https://github.com/themoeway/yomitan/raw/master/ext/data/schemas/dictionary-term-meta-bank-v3-schema.json"
+    }
+  ],
+```
+
+## Conjugation
+
+For Yomitan to conjugate Japanese terms, they need the appropriate part of speech tag. The part of speech labels are documented on the [official JMDict page](http://www.edrdg.org/jmdictdb/cgi-bin/edhelp.py?svc=jmdict&sid=#kw_pos). An [npm package](https://www.npmjs.com/package/yomichan-dict-reader) can help with extracting conjugations from JMDict, as demonstrated in the [JP-Mongolian dictionary](https://github.com/MarvNC/yomichan-dictionaries/blob/master/mongolian/scrape.js).
```

Comment:
The part of speech label for other languages can be found in `ext/js/language/{language}/{language}-transforms.js` in the `conditions` part not prefixed with "Intermediate"